flow/render_status.py

```diff
@@ -0,0 +1,220 @@
+# make separate python class for render_status
+from .util import mistune
+from .scheduling.base import JobStatus
+
+
+class renderer:
```

Comment:
Classes should start with uppercase.
```suggestion
class Renderer:
```